SoccerPlanner/app/forms.py

```diff
@@ -27,6 +28,62 @@ class Meta:
         model = User
         fields = ('username', 'first_name', 'last_name', 'email', 'password1', 'password2', )
 
+class MatchCreateForm(ModelForm):
+    team1 = forms.ModelChoiceField(queryset = Team.objects.all(), required=True, empty_label="(None)", label='Team no.1', help_text='Required')
+    team2 = forms.ModelChoiceField(queryset = Team.objects.all(), required=True, empty_label="(None)", label='Team no.2', help_text='Required') 
+    date = forms.DateField(help_text='e.g. 2019-01-01 (YYYY-MM-DD)')
+    class Meta:
+        model = Match
+        fields = ('team1','team2','date',)
+
+class PlayerCreateForm(ModelForm):
+    name = forms.CharField(max_length=20, required = True, label='Name', help_text='Required')
+    secondName = forms.CharField(max_length=20, required = True, label='Second name', help_text='Required')
+    role = forms.CharField(max_length=20, label='Role', required = False)
+    birthDate = forms.DateField(help_text='e.g. 2019-01-01 (YYYY-MM-DD)')
+    height = forms.IntegerField(label='Height (cm)', required = False, min_value=150)
+    class Meta:
+        model=Player
+        fields=('name','secondName','role','birthDate','height',)
+
+class MatchEditForm(ModelForm):
+    listOfTeams = forms.ModelChoiceField(queryset=Match.objects.all(), label="Match", required=True, empty_label="(None)")
+    date = forms.DateField(help_text='e.g. 2019-01-01 (YYYY-MM-DD)')
+    points = forms.IntegerField(label="Goals scored by team 1", required=True, min_value=0)
+    points2 = forms.IntegerField(label="Goals scored by team 2", required=True, min_value=0)
+    class Meta:
+        model=Match
+        fields=['listOfTeams', 'date', 'points', 'points2']
+
+class PlayerEditForm(ModelForm):
+    listOfPlayers = forms.ModelChoiceField(queryset = Player.objects.all(), label = "Player", required = True, empty_label ="(None)")
+    name = forms.CharField(max_length=20, required = True, label='Name')
+    secondName = forms.CharField(max_length=20, required = True, label='Second name')
+    role = forms.CharField(max_length=20, label='Role')
+    birthDate = forms.DateField(help_text='e.g. 2019-01-01 (YYYY-MM-DD)')
+    height = forms.IntegerField(label='Height (cm)', min_value=150)
+    numberOfGoals = forms.IntegerField(label='Scored Goals', min_value=0)
+    class Meta:
+        model = Player
+        fields=['listOfPlayers', 'name', 'secondName', 'role', 'birthDate', 'height', 'numberOfGoals']
+
+class PlayerDeleteForm(ModelForm):
+    listOfPlayers = forms.ModelChoiceField(queryset = Player.objects.all(), label = "Player", required = True, empty_label ="(None)")
+    class Meta:
+        model = Player
+        fields = ('listOfPlayers',)
+
+class MatchDeleteForm(ModelForm):
+    listOfMatches = forms.ModelChoiceField(queryset = Match.objects.all(), label = "Match", required = True, empty_label ="(None)")
+    class Meta:
```

Comment:
If class Meta is empty, VS screams about indentation. This should be removed.

Reply:
There should be something like 
"model = Match
fields = ('listOfMatches',)"
in Meta part, idk why it`s empty :/

Reply:
As I see, MatchDeleteForm`s Meta is not empty anymore
![loll](https://user-images.githubusercontent.com/49675793/58402100-d2e54500-805f-11e9-8fef-e452e37bae2d.png)